_posts/2018-10-01-better-ui-components.md

```diff
@@ -223,7 +222,7 @@ alert[type="success"]:before {
 
 Ok, that's starting to really look like something. (Note that the CSS here does not include all the properties needed) 
 
-It's pretty common for Alerts to disappear automatically, so let's add support for that. If there really was an HTML `alert` element and it had an auto-disappearing feature I could imagine it would have an `autodismiss` attribute for triggering this behavior, so let's go with that:
+It's pretty common for Alerts to disappear automatically, so let's add support for that. If there really was an HTML `alert` element and it had an auto-disappearing feature, imagine it with an `autodismiss` attribute for triggering this behavior, so let's go with that:
```

Comment:
you haven't used the first-person perspective yet; switching to "I" is best reserved for the conclusion.